asm/parameters.s

```diff
@@ -0,0 +1,22 @@
+#include "ultra64/asm.h"
+
+.section .text
+
+.macro IPL_SYMBOL name, address, size
+    .global \name
+    .set \name, \address
+    .type \name, @object
+    .size \name, \size
+.endm
+
+IPL_SYMBOL leoBootID,       0x800001a0, 4
```

Comment:
why move this in a new file? and why `parameters.s`

Reply:
This is where they are defined in libultra. I created this file in order to house the 0x60 bytes of 0 that used to be in bzero. With this, all the extra padding that was in the handwritten asm is accounted for properly.